testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/validation/PathSuppressionTest.java

```diff
@@ -82,7 +84,7 @@ public static void after() throws Exception {
      */
     @Test
     public void testInputSuppressPathDefault() throws Exception {
-        doTestInputViolations("default", "s", "t", "", "test.arg0");
+        doTestInputViolations("default", "s", "t", "", new String[]{"test.arg0", "test.u"});
```

Comment:
What is the reason for this change? I run this test and violation object still contains "test.arg0":
```
[PARAMETER]
[test.arg0]
[size must be between 4 and 6]
[c]
```

Btw, "u" is the name of affected parameter, but I don't know, what is the relationship between this name and violation report ...

Reply:
AFAIR, this was because of different versions of hibernate validator building messages differently

Reply:
Hmm ... so this PR changes hibernate validator version somehow? Or it was changed in some other PR?

Reply:
No, afair it's basically that by adding the <maven.compiler.parameters>true</maven.compiler.parameters> configuration, we're exposing a difference in behavior of new and old hibernate validator versions.